assemblyai/types.py

```diff
@@ -202,6 +202,11 @@ class WordBoost(str, Enum):
     high = "high"
 
 
+class RedactedAudioQuality(str, Enum):
```

Comment:
please rename to `PIIRedactedAudioQuality` so it follows the pattern of  `PIIRedactionPolicy` and `PIISubstitutionPolicy`. Also, the type has to be added in the `__init__.py`

Reply:
Done.